source/slang/slang-capabilities.capdef

```diff
@@ -397,3 +448,83 @@ alias DX_6_4 = sm_6_4;
 alias DX_6_5 = sm_6_5;
 alias DX_6_6 = sm_6_6;
 alias DX_6_7 = sm_6_7;
+
+alias sm_2_0_GLSL_140 = sm_4_0 | glsl | spirv_1_0 | cuda | cpp;
+alias sm_2_0_GLSL_400 = sm_4_0 | glsl | spirv_1_0 | cuda | cpp;
+alias appendstructuredbuffer = sm_5_0 + shader_stages_fra_com;
+alias atomic_hlsl = _sm_4_0 | GLSL_430_SPIRV_1_0;
+alias atomic_hlsl_nvapi = _sm_4_0 + hlsl_nvapi;
+alias atomic_hlsl_sm_6_6 = _sm_6_6;
+alias byteaddressbuffer = sm_4_0;
+alias byteaddressbuffer_rw = sm_4_0 + shader_stages_fra_com;
+alias consumestructuredbuffer = sm_5_0 + shader_stages_fra_com;
+alias fragmentprocessing = fragment + sm_5_0;
+alias fragmentprocessing_derivativecontrol = fragment + sm_5_0 + GL_ARB_derivative_control;
+alias getattributeatvertex = fragment + _sm_6_1 | fragment + GL_EXT_fragment_shader_barycentric;
```

Comment:
Wouldn't it be better with underscore characters in the names?
```
append_structured_buffer
consume_structured_buffer
fragment_processing
get_attribute_at_vertex
```

Reply:
I am open for any suggestions on naming and welcome them. the existing naming scheme avoided `_`,  so I decided to just have a placeholder style which can be changed later and made something which could easily be regex'ed for modification.